docs/tutorial/force_calibration.rst

```diff
@@ -0,0 +1,95 @@
+Force calibration
+=================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+Force calibration is used to convert the raw voltages that Bluelake records to actual forces.
+In this tutorial, we will be redoing a calibration that we already performed in Bluelake, to provide an example of using the force calibration module::
+
+    f = lk.File("calibration.h5")
+
+In this tutorial, we want to reproduce calibration item 1.
+We grab the chunk of data that was used for the calibration last time::
+
+    start = f.force1x.calibration[1]["Start time (ns)"]
+    stop = f.force1x.calibration[1]["Stop time (ns)"]
+    force_slice = f.force1x[start:stop]
+
+To be able to calibrate our forces, we first convert back our data to raw voltages.
+We can do this using the previous calibration performed in Bluelake (in this case, the first one in the file)::
+
+    volts = force_slice.data / f.force1x.calibration[0]["Rf (pN/V)"]
```

Comment:
You should also subtract the offset. And probably also mention that the offset is not calculated by the force calibration procedure. Instead, it's a user-defined zero level.

Reply:
Welp! That's a good one! I would've totally forgotten about that.